docs/refguide/map.md

```diff
@@ -1,17 +1,18 @@
 # Observable Maps
 
-`observable(asMap(values?, modifier?))` (and `map(values?, modifier?)`) creates a dynamic keyed observable map.
+## `observable.map(values)`
+
+`observable.map(values?)` creates a dynamic keyed observable map.
```

Comment:
It probably makes sense to add 

> To create dynamically keyed objects always use maps!

here as well. You can never have too few warnings :)